http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -168,13 +168,6 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         )
         .run(authContext)
 
-      // Create relay hybrid connection pool
```

Comment:
looks like it was previously being created twice, this is already done on line 140

Reply:
weird!

Reply:
yeah I had to look at it several times, but its the exact same call as above...